domain/src/jwt/mod.rs

```diff
@@ -69,15 +71,15 @@ pub async fn generate_collab_token(
         Error {
             source: None,
             // TODO make this InternalErrorKind::ConfigError
```

Comment:
```suggestion
```

Reply:
Good call. Updated in dc80f8bc14f61044a4246d9a4925302489f05bb4 !